docs/RELEASE.md

```diff
@@ -0,0 +1,175 @@
+# Release process
+
+The release process for nobl9-go cannot be discussed without the context of
+our platform releases. In the following document, `n9` will refer to our
+platform **production** version.
+
+## The process
+
+1. Create release candidate version. \
+  Most likely this step is where you can stop.
+  Creating an official version release is only necessary if you want our users
+  to have immediate access to the released changes.
+  In many scenarios it might not be the case.
+  Furthermore, creating release candidate version instead of official release
+  gives us more time to "sit" on the introduced changes, which in turn
+  increases a chance for bug detection and gives room to introduce breaking
+  changes which might be necessary to address the uncovered issue.
+2. Create official release version (restricted access). \
```

Comment:
clarify restricted access, you can be transparent that this path is only for Nobl9ers

Reply:
maybe the whole document should be divided in two sections, the first meant to open source contributors, the second for Nobl9ers

Reply:
I'm not sure we need to mention open source contributors in this README at all right now we don't even have contribution guidelines yet